spec/views/pages/delete.html.erb_spec.rb

```diff
@@ -39,4 +45,239 @@
       expect(rendered).not_to have_css ".govuk-hint"
     end
   end
+
+  describe "when page to delete is not associated with any routes" do
+    it "does not render a notification banner" do
+      expect(rendered).not_to have_css "govuk-notification-banner"
```

Comment:
I think this is missing a dot which means it'll always pass:

```suggestion
      expect(rendered).not_to have_css ".govuk-notification-banner"
```

Reply:
Ah, very good spot, thank you! This should be fixed now.